functions/index.js

```diff
@@ -250,11 +251,15 @@ exports.callNextOnQueue = functions.https.onCall(async (data, context) => {
       }
     );
   } else if (!!result.ticket.phone) {
+    let message;
+    if (!result.queue.language || result.queue.language === "pt") {
```

Comment:
This logic should be inverted. Since PT is the fallback language, this would be more suitable:
```ts
if (result.queue.language && result.queue.language === "en") {
  // EN message logic here
} else {
  // Fallback message logic here
}
```